### DISCUSSION TITLE
Can I use not-escaped HTML code inside the front matter?

### DISCUSSION THREAD

**Discussion by francoscarpa** [11ty/eleventy#2061] [Q&A] [answered]
Using the following front matter:
```
---
// Other elements
h2: "I’m <a href='#'>28</a>"
---
```
Eleventy escapes the HTML tags, and prints `I’m <a href='#'>28</a>` to the screen, because the generated source is `I’m a &lt;a href=&#39;#&#39;&gt;28&lt;/a&gt;`. Can I make Eleventy so that it doesn’t escape HTML code inside the front matter?

**Comment by pdehaan** [comment, marked as answer]
Assuming you’re using nunjucks, try `{{ h2 | safe }}`

**Reply by isralduke** [reply]
thank you
